src/databricks/labs/ucx/queries/progress/main/01_00_percentage_migration_progress.sql

```diff
@@ -2,4 +2,4 @@
 SELECT
     ROUND(100 * try_divide(COUNT_IF(SIZE(failures) = 0), COUNT(*)), 2) AS percentage
 FROM ucx_catalog.multiworkspace.objects_snapshot
-WHERE object_type IN ('ClusterInfo', 'Grant', 'JobInfo', 'PipelineInfo', 'PolicyInfo', 'Table', 'Udf')
+WHERE object_type IN ('ClusterInfo', 'DirectFsAccess', 'Grant', 'JobInfo', 'PipelineInfo', 'PolicyInfo', 'QueryProblem', 'Table', 'Udf', 'UsedTable')
```

Comment:
directfs access, query, usedtable - they're not top-level entities per se, but jobs, pipelines, dashboards - are